cpp/include/cudf/column/column_device_view.cuh

```diff
@@ -472,6 +472,13 @@ class alignas(16) column_device_view : public detail::column_device_view_base {
     return d_children[child_index];
   }
 
+  /**
+   * @brief Returns the number of child columns
+   *
+   * @return size_type The number of child columns
```

Comment:
```suggestion
   * @return The number of child columns
```

Reply:
should this be for all doxygen comments? There are plenty of such occurrences in our codebase.

Reply:
Yes. The return type should not be included. 
https://github.com/rapidsai/cudf/blob/branch-0.19/cpp/docs/DOCUMENTATION.md#return
There is apparently a vscode plugin that some are using that is adding these into their code.

Reply:
Right. Adding this to settings.json would help. (rapids-compose also uses it)
`    "doxdocgen.generic.returnTemplate": "@return ",`